src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -27,10 +28,12 @@ export class ExerciseChatWidgetComponent implements OnInit, OnDestroy, AfterView
     newMessageTextContent = '';
     isLoading: boolean;
     sessionId: number;
+    error: IrisErrorType | null;
```

Comment:
Usually we use undefined instead of null. Is there a reason why you use null here?

Reply:
AFAIK undefined should be used when there is no previous value. In this case there might be an error, or none. Null represents the second case.